fudge-ratio/src/main/webapp/calculator.html

```diff
@@ -48,5 +48,5 @@ <h1>Fudge Ratio Calculator</h1>
 
         <a href="ActivitiesManager.html"> Go to activities Manager </a>
     </body>
-    <script defer src="script.js"></script>
+    <script defer src="main.js"></script>
```

Comment:
Again, as we are using defer, it’s safe to just run the command, because we are guaranteed that all the DOM elements already exist